### ISSUE TITLE
RFTools storage scanner power drain

### ISSUE THREAD

**Issue by inviktus** [McJtyMods/RFTools#1837]
I'm playing ATM3 for 1.12.2. I was experimenting with the storage scanner and xnet to see if they would work together somehow. When I have an xnet network/connector attached to the storage scanner, I notice it drains all the power on my power grid. I'm unsure if this is a rftools or xnet issue, so apologies if I've posted this on the wrong tracker.

This was in a new world and very early game so the only power I have right now is a couple of water wheels from immersive engineering.

I had only configured item channels via the controller if that is relevant (no power channels)

Steps to repeat,
attach xnet network/connector to rftools storage scanner, watch it drain all your power.

**Comment by McJty** [comment]
XNet indeed constantly drains power. That's how it is supposed to be

**Comment by inviktus** [comment]
Even if no power channels are configured? The xnet controller attached to the same powergrid does not drain my entire grid, only when the xnet network is attached to the storage scanner does my grid lose all power.

**Comment by McJty** [comment]
I have to look it up. Can you show me a screenshot of your setup?

**Comment by inviktus** [comment]
Didn't actually mean to close this.

**Comment by inviktus** [comment]
Yep, I get that. It's not the xnet controller that is the problem though. As I said before, it's when the rftools storage scanner is connected to an xnet network, it drains everything, instantly.

**Comment by McJty** [comment]
Was it an empty storage scanner? I mean the storage scanner has an internal buffer that it needs to fill. It will do that as soon as possible. Once it is full that should no longer happen though

**Comment by inviktus** [comment]
No the storage scanner RF buffer was full when I attempted to connect it to xnet, as it had been in use for some time previously.

Once I get around to better power generation, I'll test again and let you know what happens, though it doesn't seem like the storage scanner should drain all power instantly and ongoing when I'm supplying 528 RF/t
